monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js

```diff
@@ -515,13 +516,15 @@ class ReportPageComponent extends AuthComponent {
 
   generateIssues = (issues) => {
     let issuesDivArray = [];
-    for (let machine of Object.keys(issues)) {
+    for (let machineId of Object.keys(issues)) {
+      let machine = getMachineFromID(parseInt(machineId), this.state.machines);
+      let hostname = getMachineHostname(machine);
       issuesDivArray.push(
-        <li key={JSON.stringify(machine)}>
-          <h4><b>{machine}</b></h4>
+        <li key={machineId}>
+          <h4><b>{hostname}</b></h4>
```

Comment:
This is missleading as it might be IP.

Reply:
I agree. It should be something like `display_name`. However, this hostname/IP thing is in a bunch of places in the codebase, so again, I'd say we put it off until to when we refactor the report.